### ISSUE TITLE
Collect references for this project

### ISSUE THREAD

**Issue by malvikasharan** [alan-turing-institute/data-training-for-bioscience#1]
# Summary

List any relevant documents or resources that will be useful in the development of this project

## Turing resources

- The Turing Way materials: https://the-turing-way.netlify.app/welcome
- RSE4DataScience: https://github.com/alan-turing-institute/RSE4DataScience18
- RDS course - under development: https://github.com/alan-turing-institute/rds-course
- Data Science Skills: https://github.com/alan-turing-institute/DataScienceSkills
- RSE Summer School: https://github.com/alan-turing-institute/DH-RSE-Summer-School

## Carpentries Resources
- Genomics Lessons: https://datacarpentry.org/lessons/#genomics-workshop
-  Ecology Lessons: https://datacarpentry.org/lessons/#ecology-workshop

## Crick institute
- Request for links to materials or resources for reuse

**Comment by malvikasharan** [comment]
- RDA-CODATA Summer School: https://github.com/CODATA-RDA-DataScienceSchools (Check with Patricia H. and DCC)
- Check with Life Science training network

**Comment by malvikasharan** [comment]
EMBL-EBI resources: https://competency.ebi.ac.uk/

**Comment by Lisanna** [comment]
I think this issue (and task) should be divided into two lists of references. 
1) Landscape analysis: what is the current offer for someone that looks for training in Data Science? What pops up when you Google for it? And in YouTube? Which are the search keywords?
This should be done before starting to develop materials and even deciding the specific topics, in my opinion.
2) General references, collected and used during the course development, linked also to the project partners. Most of the links here belong to this second list.

**Comment by malvikasharan** [comment]
Good point! I definitely am mixing them both here. Landscape analysis can be a combination of resources we would want to use as well as examples of resources that are not effectively used (or that are not designed well) by the new learners with bioscience backgrounds.

**Comment by Lisanna** [comment]
[Open, Data-driven Science for Decision Makers](https://github.com/JasonJWilliamsNY/openscience_for_decisionmakers)

**Comment by malvikasharan** [comment]
https://twitter.com/EBItraining/status/1427197636730306563?s=20

**Comment by malvikasharan** [comment]
Cambridge courses: 
- https://training.csx.cam.ac.uk/bioinformatics/course/bioinfo-python/
- Data science in Python: https://training.csx.cam.ac.uk/bioinformatics/event/3748813

**Comment by malvikasharan** [comment]
Shared by @fedenanni:
https://alan-turing-institute.github.io/rds-course/modules/m1/1.1-WhatIsDataScience.html
